server/Controllers/RecipesController.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using CookBook.API;
+using CookBook.API.Requests;
+using CookBook.Services;
+using CookBook.API.Requests.External;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CookBook.Controllers
+{
+    [ApiController]
+    public class RecipesController : Controller
+    {
+        private readonly IRecipeRepository _recipeRepo;
+
+        public RecipesController(IRecipeRepository recipeRepo)
+        {
+            _recipeRepo = recipeRepo;
+        }
+
+        [HttpGet("/recipePriceBreakdown/{id}")]
+        public async Task<IActionResult> Get(long id)
+        {
+            var model = await _recipeRepo.GetRecipePriceBreakdown(id);
+
+            if (model == null)
+                return Forbid(); //or any other error code accordingly. Bad request is a strong candidate also.
```

Comment:
raczej NotFound()

```suggestion
                return NotFound(); //or any other error code accordingly. Bad request is a strong candidate also.
```

Reply:
Zrobiłem jakieś tam rzucanie wyjątku, jak bede juz pushowal to sprawdzisz i ewentualnie poprawie.